lib/docbuild.js

```diff
@@ -0,0 +1,7 @@
+createDocs = require('node-offline-api').createDocs
+buildOptions = require('node-offline-api').buildOptions
```

Comment:
can we use something like `var buildOptions = require('')` or `const` may be.

That would help the test to pass.